### PR TITLE
Add ability to specify which github stats to return

### PR DESCRIPTION
**Pull request by cansavvy**
#### What changes are being implemented in this Pull Request?
@kweav and I pair programmed on this. It is meant to address #88 

We had to carry over the changes to a number of the arguments but it should be a bit cleaner now. People can specify a vector with what stats they'd like collected. 

#### What was your approach?

We tested this interactively using the examples (and added some examples to the docs too) 

#### What GitHub issue does your pull request address?

#88

### Tell potential reviewers what kind of feedback you are soliciting.

@kweav and I might look at this with fresh eyes later just to see if there's anything else that could use polishing.

### CODE REVIEW

File: R/github.R [hunk 1]
```diff
@@ -400,29 +458,46 @@ clean_repo_metrics <- function(repo_name, repo_metric_list) {
...
+  }
+
+  clean_stats_names <- list(
+    repo_activity = "num_repo_activities",
+    contributors = c("num_contributors","total_contributors"),
+    forks = "num_forks",
+    stars = "num_stars",
+    community = "health_percentage")
+
+  #
```
Comment (cansavvy) [inline R/github.R:495]:
```suggestion
  # We're going to only put the stats we collected in the data.frame
```

**Comment by cansavvy** [comment]
@kweav I feel pretty good about this code but you take a look and see if there's anything else we should improve. Tests are passing so feel free to approve/merge as you'd like.

File: R/github.R [hunk 2]
```diff
@@ -400,29 +458,46 @@ clean_repo_metrics <- function(repo_name, repo_metric_list) {
...
+  } else {
+    cleaned_metrics$num_forks <- NA
+  }
+
+  if (is.list(repo_metric_list$stars)) {
+    # GitHub API returns a list of who's given a star we just want a number
+    cleaned_metrics$num_stars <- length(unlist(purrr::map(repo_metric_list$stars, "login")))
   } else {
-    num_forks <- NA
+    cleaned_metrics$num_stars <- NA
```
Comment (kweav) [inline R/github.R:479]:
```suggestion
    cleaned_metrics$num_forks <- NA
```

**Review by kweav — APPROVED** [review]
Looks great! I like the comments a lot. They were a good reminder of what the code was doing

**Pull request merged**